metron-streaming/Metron-Pcap_Service/pom.xml

```diff
@@ -114,15 +114,24 @@
 			<version>2.3</version>
 		</dependency>
 		<dependency>
+			<groupId>com.google.guava</groupId>
+			<artifactId>guava</artifactId>
+			<version>${global_hbase_guava_version}</version>
+		</dependency>
+		<dependency>
 			<groupId>org.apache.hbase</groupId>
 			<artifactId>hbase-client</artifactId>
 			<version>${global_hbase_version}</version>
-			<scope>provided</scope>
+			<!--scope>provided</scope-->
```

Comment:
Rather than comment this out, can you remove it? Unless you think it should stay.